MoppApp/MoppApp/ContainerViewController.swift

```diff
@@ -289,18 +233,22 @@ extension ContainerViewController : UITableViewDataSource {
     }
     
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        guard let container = container else {
+        if (containerViewDelegate.isContainerEmpty()) {
```

Comment:
Outer parenthesis in `if` statements should be removed for overall style consistency.
(It would be ok if there is parentheses used everywhere in such a way)